packages/rpc-core/src/rpc-methods/getGenesisHash.ts

```diff
@@ -0,0 +1,13 @@
+import { Base58EncodedAddress } from '@solana/addresses';
+
+type GetGenesisHashApiResponse = Base58EncodedAddress;
```

Comment:
We should probably have a different type for this. It's not actually an address, so we don't want values of this kind to typecheck in places where an address is expected.

Reply:
Would this be a blockhash instead? Genesis has is a blockhash right?

Reply:
Oh, maybe! I guess that's it.